frontend/src/i18n/zh-cn.json

```diff
@@ -90,8 +90,8 @@
     "copy": "复制",
     "copyMessage": "请选择欲复制至的目录：",
     "currentlyNavigating": "当前目录：",
-    "deleteMessageMultiple": "你确定要删除这 {count} 个文件吗?",
-    "deleteMessageSingle": "你确定要删除这个文件/文件夹吗?",
+    "deleteMessageMultiple": "你确定要删除这 {count} 个文件吗？",
+    "deleteMessageSingle": "你确定要删除这个文件/文件夹吗？",
```

Comment:
Why did you add a space at the end?

Reply:
Thanks for your comment.  That is not spaces, but special characters called Fullwidth Characters in CJK computing, which helps us read characters more easily when combining with Chinese characters. Furthermore, it is the common way we use in strings having Chinese. Please see [Halfwidth and fullwidth forms](https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms) for more information.

e.g.
"" and “”
() and （）
! and ！
. and 。

See the differences?